powershell/admin-center/Microsoft.PowerApps.Administration.PowerShell.Samples.psm1

```diff
@@ -1665,6 +1665,127 @@ function SetManagedEnvironmentSolutionCheckerEnforcementLevel
     Write-Host "Set solution checker enforcement for the specified environment."    
 }
 
+function SetManagedEnvironmentMakerOnboardingMarkdownContents
+{
+    <#
+     .SYNOPSIS
+     Sets markedown contents for maker onboarding for the specified Managed environment.
+     .DESCRIPTION
+     The SetManagedEnvironmentMakerOnboardingMarkdownContents cmdlet sets markedown contents for maker onboarding for the specified Managed environment.
+     Use Get-Help SetManagedEnvironmentMakerOnboardingMarkdownContents -Examples for more details.
+     .PARAMETER EnvironmentId
+     The id (usually a GUID) of the environment.
+     .PARAMETER Markdown
+     The maker contents Markdown.
+     .EXAMPLE
+     SetManagedEnvironmentMakerOnboardingMarkdownContents -EnvironmentId 8d996ece-8558-4c4e-b459-a51b3beafdb4 -Markdown "## Welcome to Power Apps
```

Comment:
Hmm you should try to test this locally to make sure you can set a multiline string parameter like this.
https://stackoverflow.com/questions/31793449/how-to-assign-multiple-lines-string-in-powershell-console